ApeWorX/ape
# feat!: ecosystem decode and encode address

BREAKING CHANGE: Adds a new abstract method and deletes `utils` method `to_address`.

### What I did

fixes: #579 

Allow ecosystem API to decode and encode addresses

### How I did it

new abstract method

### How to verify it

See test
Also going to make sure Starknet can do its thang. I wll move out of draft then.

### Checklist
